controllers/routemonitor_controller.go

```diff
@@ -142,14 +142,19 @@ func (r *RouteMonitorReconciler) deploymentForBlackboxExporter(m *monitoringv1al
 
 // deploymentForServiceMonitor returns a ServiceMonitor
 func (r *RouteMonitorReconciler) deploymentForServiceMonitor(m *monitoringv1alpha1.RouteMonitor) *monitoringv1.ServiceMonitor {
-	var ls metav1.LabelSelector = metav1.Convert_Map_string_To_string_To_v1_LabelSelector(labelsForRouteMonitor(m.Name), &metav1.LabelSelector{}, nil)
+	var ls *metav1.LabelSelector
+	routeMonitorLabels := labelsForRouteMonitor(m.Name)
+	err := metav1.Convert_Map_string_To_string_To_v1_LabelSelector(&routeMonitorLabels, ls, nil)
+	if err != nil {
+		panic(1)
```

Comment:
yikes! Isn't there a more graceful way or at least inform why we panic?

Reply:
you are correct, I just wanted to make the build pass quickly, will fix XD